Cubical/Foundations/GroupoidLaws.agda

```diff
@@ -0,0 +1,93 @@
+{-
+
+This file proves the higher groupoid structure of types
+
+-}
+{-# OPTIONS --cubical --safe #-}
+module Cubical.Foundations.GroupoidLaws where
+
+open import Cubical.Core.Primitives
+open import Cubical.Core.Prelude
+
+private
+  variable
+    ℓ : Level
+    A : Set ℓ
+
+-- some useful notation
+
+_·_ :  {x y z : A} → (x ≡ y) → (y ≡ z) → (x ≡ z)
+x≡y · y≡z = compPath x≡y y≡z 
+
+infixr 30 _·_
+
+_⁻¹ : {x y : A} → (x ≡ y) → (y ≡ x)
+x≡y ⁻¹ = sym x≡y
+
+𝟭 : {x : A} → x ≡ x
```

Comment:
I think I suggested adding this, maybe that was a mistake. How about changing it back to refl?

Reply:
Yeah, that's true --- I will do that in my next commit.